src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -619,6 +620,13 @@ private Link handleUnknownLink(de.tudarmstadt.ukp.wikipedia.parser.Link link)
 		}
 	}
 
+
+    /**
+	 * Checks if provided link has an image pattern as target. Some links that get extracted from the <gallery></gallery>
```

Comment:
Pretty sure that `<gallery></gallery>` won't show up in the generated Javadoc. Can someone with this checked out and working in an IDE check please?